src/tinyspline.h

```diff
@@ -5,6 +5,22 @@
 
 #include <stddef.h>
 
+#ifdef _WIN32
+# define TINYSPLINE_SHARED_EXPORT __declspec(dllexport)
+# define TINYSPLINE_SHARED_IMPORT __declspec(dllexport)
+#else
+# define TINYSPLINE_SHARED_EXPORT
+# define TINYSPLINE_SHARED_IMPORT
+#endif
+
+#ifdef TINYSPLINE_STATIC
```

Comment:
@msteinbeck 
This is the reason for `TINYSPLINE_STATIC`

When building a shared library on Windows using MSVC, you need to explicitly tag the symbols that you want to export. When building a static library, you generally don't want to export the symbols and just keep default visibility.

To compare with gcc, MSVC has always `-fvisibilty=hidden` enabled.

There is a similar change in `src/tinysplinecxx.h`.

Reply:
Thanks for clarification.

Reply:
There is an albernative to my approach:
1. generate an export header using https://cmake.org/cmake/help/latest/module/GenerateExportHeader.html for each library
2. use these defines in the headers

But this will generate 2 extra headers (one for tinyspline and one for tinysplinecxx) which I felt was too many for a small library.

Reply:
> But this will generate 2 extra headers (one for tinyspline and one for tinysplinecxx) which I felt was too many for a small library.

I agree.